src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <img
+        src="images/Search.svg"
+        alt="magnifier"
+        class="big-img"
+      >
+      <input
+        type="text"
+        class="input big-input"
+        name="big-serch-bar"
+        data-qa="keypress"
+        placeholder="Try &#34Los Angeles&#34"
+      >
+    </form>
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+      class="form"
+    >
+      <img
```

Comment:
The 'img' elements should be self-closing. You can close it by adding a '/' before the '>' like this: <img />.